### PR TITLE
Stored the Random object in a private field, also did some formatting #11

### PR DESCRIPTION
**Pull request by BGergo03**
#11

### CODE REVIEW

File: src/main/java/DrukmakoriSivatag/Random.java [hunk 1]
```diff
@@ -9,73 +8,77 @@
 public class Random {
 
     /**
-     *  Egy random generált érték alapján eldönti, hogy elromoljon-e a pumpa, vagy sem, majd visszatér a döntéssel
+     * Random object for generating random numbers
+     */
+    private final java.util.Random myRandom = new java.util.Random();
```
Comment (andrasig-bme) [inline src/main/java/DrukmakoriSivatag/Random.java]: Using a pseudorandom generator is not safe, try using another random generator instead.

**Review by andrasig-bme — CHANGES_REQUESTED** [review]

**Review by forriM — APPROVED** [review]

File: src/main/java/DrukmakoriSivatag/Random.java [hunk 2]
```diff
@@ -9,73 +9,77 @@
 public class Random {
 
     /**
-     *  Egy random generált érték alapján eldönti, hogy elromoljon-e a pumpa, vagy sem, majd visszatér a döntéssel
+     * Random object for generating random numbers
```
Comment (andrasig-bme) [inline src/main/java/DrukmakoriSivatag/Random.java]: Comments on other part of the code are in Hungarian.

**Review by andrasig-bme — CHANGES_REQUESTED** [review]
Changes of the code are ok, please fix the marked comment to match the language of the other comments.

**Review by andrasig-bme — APPROVED** [review]
Well done.

**Pull request merged**